# Sets the correct access attributes for register lists of VSCCLRM instructions.
# See issue: https://github.com/llvm/llvm-project/issues/62455

diff --git a/arch/ARM/ARMGenCSMappingInsnOp.inc b/arch/ARM/ARMGenCSMappingInsnOp.inc
index a3b04d915..12c9a1f9c 100644
--- a/arch/ARM/ARMGenCSMappingInsnOp.inc
+++ b/arch/ARM/ARMGenCSMappingInsnOp.inc
@@ -70562,7 +70562,7 @@
 	      CS_AC_READ,
 	      { CS_DATA_TYPE_i32, CS_DATA_TYPE_LAST } }, /* p - i32imm */
 	    { CS_OP_IMM,
-	      CS_AC_READ,
+	      CS_AC_WRITE,
 	      { CS_DATA_TYPE_i32, CS_DATA_TYPE_LAST } }, /* regs */
 	    { CS_OP_INVALID, CS_AC_READ, { CS_DATA_TYPE_LAST } }, /*  */
 	    { 0 } }
@@ -70575,7 +70575,7 @@
 	      CS_AC_READ,
 	      { CS_DATA_TYPE_i32, CS_DATA_TYPE_LAST } }, /* p - i32imm */
 	    { CS_OP_IMM,
-	      CS_AC_READ,
+	      CS_AC_WRITE,
 	      { CS_DATA_TYPE_i32, CS_DATA_TYPE_LAST } }, /* regs */
 	    { CS_OP_INVALID, CS_AC_READ, { CS_DATA_TYPE_LAST } }, /*  */
 	    { 0 } }
